include/expand_list.hpp

```diff
@@ -0,0 +1,200 @@
+#include <tune_quda.h>
+#include <int_factor_array.hpp>
+
+namespace quda {
+
+/**
+    @brief Thie helper class instantiates the following mapping:
```

Comment:
`Thie` -> `This`

Reply:
https://github.com/lattice/quda/pull/1497/commits/6451e7fb56974b902baacdb64f547ef8eb5231c4